BBPebbleLib/Pebble.cpp

```diff
@@ -38,46 +43,86 @@ void Pebble::onDisconnect()
 
 void Pebble::onDataReceived(QBluetoothSocket &bt_socket)
 {
-    QDataStream in(&bt_socket);
-
-    if (this->payloadSize == 0) {
-        if (bt_socket.bytesAvailable() < (int)2*sizeof(quint16)) {
+    static const int header_length = (int)2*sizeof(quint16);
```

Comment:
Right, much easier to read, however we could double check if it is the proper naming convention for a static const variable within a fucntion.

Also, the library being built on top of Qt, I would rather using Qt type than C++ type if you don't mind :) Samething below :)